src/jit/gentree.cpp

```diff
@@ -12660,8 +12660,8 @@ GenTree* Compiler::gtFoldTypeCompare(GenTree* tree)
         // If we have class handles, try and resolve the type equality test completely.
         if ((cls1Hnd != nullptr) && (cls2Hnd != nullptr))
         {
-            JITDUMP("Asking runtime to compare %p (%s) and %p (%s) for equality\n", cls1Hnd,
-                    info.compCompHnd->getClassName(cls1Hnd), cls2Hnd, info.compCompHnd->getClassName(cls2Hnd));
+            JITDUMP("Asking runtime to compare %p (%s) and %p (%s) for equality\n", dspPtr(cls1Hnd),
+                    info.compCompHnd->getClassName(cls1Hnd), dspPtr(cls2Hnd), info.compCompHnd->getClassName(cls2Hnd));
```

Comment:
This change and the one in importer.cpp are unrelated, except that they make it a lot easier to examine diffs in jitdumps.